### PR TITLE
Fix SPS vertexdata serialization

### PR DESCRIPTION
**Pull request by a-carvallo**

### CODE REVIEW

File: src/Particles/babylon.solidParticleSystem.ts [hunk 1]
```diff
@@ -200,10 +200,10 @@
                 vertexData.indices = (this._depthSort) ? this._indices : this._indices32;
                 vertexData.set(this._positions32, VertexBuffer.PositionKind);
                 vertexData.set(this._normals32, VertexBuffer.NormalKind);
-                if (this._uvs32) {
+                if (this._uvs32.length > 0) {
```
Comment (deltakosh) [inline src/Particles/babylon.solidParticleSystem.ts]: What if uv32 is null or undefined?

Comment (a-carvallo) [reply src/Particles/babylon.solidParticleSystem.ts]: Fair point, a few lines before that, there is this line : 
```ts
this._uvs32 = new Float32Array(this._uvs);
```
So it should never be null or undefined.

Comment (deltakosh) [reply src/Particles/babylon.solidParticleSystem.ts]: Makes sense:)
Please fix the conflict and I can merge

Comment (a-carvallo) [reply src/Particles/babylon.solidParticleSystem.ts]: Should be good !

**Comment by deltakosh** [comment]
Thanks!

**Pull request merged**